codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -959,4 +940,139 @@ private boolean isLocal() {
             return this == localEndpoint;
         }
     }
+
+    /**
+     * Provides an interface to manage the active streams for this {@link Http2Connection}.
+     */
+    private interface ActiveStreams {
```

Comment:
@buchgr - Was thinking about that, but we are using `this` in the constructor which is also awkward...not that we are necessarily avoiding the issue here but it is still not ideal....I'll make the change and commit to see how it looks.